operator/src/test/java/oracle/kubernetes/operator/helpers/ManagedPodHelperTest.java

```diff
@@ -1081,11 +1079,11 @@ void whenDomainAndClusterHaveLegacyAuxImages_createManagedPodsWithInitContainers
     assertThat(getCreatedPodSpecInitContainers(),
         allOf(Matchers.hasLegacyAuxiliaryImageInitContainer(AUXILIARY_IMAGE_INIT_CONTAINER_NAME_PREFIX + 1,
                 "wdt-image:v1",
-                V1Container.ImagePullPolicyEnum.IFNOTPRESENT,
+                        "IfNotPresent",
```

Comment:
Indentation not the same as below?

Reply:
My IDE did this automatically... I'll see if I can catch these but not sure that it matters.